packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -164,6 +165,17 @@ export default {
       type: Array,
       required: true
     },
+    /**
```

Comment:
the doc-block here is not needed, the table is files-app only and kinda "private"

Reply:
I think we discussed you guys would refactor what was necessary and merge this PR (or a replacement of it). Or do you need us to do anything?

Reply:
We can take care. Looking at this PR, I have the impression that the grouping settings for the shared with me page will stay in your fork of web until you have it bulletproof? Asking because this PR only adds a property for the ResourceTable and applies it to the ODS table.

Reply:
@elizavetaRa what happened to the configs? Since we now have the same shared with me page, it's ok to have everything merged upstream.